include/kf/EarlyAllocator.h

```diff
@@ -24,42 +24,44 @@ namespace kf
 
         template<class Y>
         constexpr EarlyAllocator(const EarlyAllocator<Y>& other) noexcept : m_ptr(reinterpret_cast<T*>(other.m_ptr)), m_size(other.m_size)
-        {
-        }
+        { }
 
-        _CONSTEXPR20_DYNALLOC ~EarlyAllocator() = default;
-        _CONSTEXPR20_DYNALLOC EarlyAllocator& operator=(const EarlyAllocator&) = default;
+        ~EarlyAllocator() = default;
+        EarlyAllocator& operator=(const EarlyAllocator&) = default;
 
         template<POOL_TYPE poolType>
         T* initialize(const size_t count)
         {
-            assert(!m_ptr);
```

Comment:
Why do we need this change?

Reply:
I had some problems in building with assert instead of ASSERT

Reply:
Changed to _Xinvalid_argument